Source/WebCore/rendering/RenderBlock.h

```diff
@@ -39,7 +39,7 @@ class RenderText;
 struct BidiRun;
 struct PaintInfo;
 
-typedef ListHashSet<RenderBox*> TrackedRendererListHashSet;
+typedef WeakListHashSet<RenderBox> TrackedRendererListHashSet;
```

Comment:
`using`

Also not a great type name. Not sure why a set type should be named a "tracked list" even if the set is used to "track a list of things".